Cql/Cql.Compiler/LibraryDefinitionBuilderSettings.cs

```diff
@@ -46,12 +46,12 @@ internal class LibraryDefinitionBuilderSettings
 
         /// <summary>
         /// When <see langword="true"/>, functions declared external will throw <see cref="NotImplementedException"/> at runtime
-        /// if they are not found in <see cref="ExpressionBuilder._customImplementations"/>.  When <see langword="false"/>, an
-        /// <see cref="InvalidOperationException"/> will be thrown during compilation if they are not found in <see cref="ExpressionBuilder._customImplementations"/>.
+        /// if they are not found in customImplementations.  When <see langword="false"/>, an
```

Comment:
So, in the end probably custom implementations (adding your own hand-written functions to a library) is probably the preferred mechanism to override existing functions, or supply the implementation of externals?

Reply:
I see not this was a public property on ExpressionBuilder, and even though there are no examples of its usage in the repo, I'll create a task to add this back. Probably better to put this in the settings itself.